pages/polling/[poll-hash].tsx

```diff
@@ -64,6 +69,32 @@ const PollView = ({ poll }: { poll: Poll }) => {
   const { tally } = usePollTally(poll.pollId, 60000);
   const { comments, error: errorComments } = usePollComments(poll.pollId);
 
+  useEffect(() => {
+    const fetchFilteredPolls = async categories => {
+      const { polls: pollData } = await fetchJson(
```

Comment:
i suggest we move this fetchJSON to a useSWR hook, so we can avoid refetching (it will use swr internal cache) if the query filter does not change.

Reply:
Good call, I added the hook.